src/utils.js

```diff
@@ -136,7 +136,30 @@ export const isPruneTraceOff = () =>
   );
 
 export const isSwitchedOff = () => {
-  return TracerGlobals.getTracerInputs().switchOff;
+  return TracerGlobals.getTracerInputs().switchOff || !isValidAlias();
+};
+
+export const getValidAliasesOrEmptyArray = () =>
+  process.env['VALID_ALIASES'] || [];
```

Comment:
Talk with Efi about the name, maybe should be
```suggestion
  process.env['LUMIGO_VALID_ALIASES'] || [];
```